device/mka/basic.yaml

```diff
@@ -0,0 +1,262 @@
+        send_icv_indicatior_in_mkpdu:
+          description: |-
+            Send ICV Indicator in MKPDU.
+          type: boolean
+          default: true
+          x-field-uid: 10
+        delay_protect:
+          description: |-
+            Delay Protect.
+          type: boolean
+          default: true
+          x-field-uid: 11
+        rekey_mode:
+          description: >-
+            Rekey Mode.
+          x-field-uid: 12
+          $ref: '#/components/schemas/Mka.Basic.RekeyMode'
+
+    Mka.Basic.SupportedCipherSuites:
+      description: >-
+        The container for supported cipher suites.
+      type: object
+      properties:
+        gcm_aes_128:
+          description: |-
+              GCM-AES-128.
+          type: boolean
+          default: true
+          x-field-uid: 1
+        gcm_aes_256:
+          description: |-
+              GCM-AES-256.
+          type: boolean
+          default: true
+          x-field-uid: 2
+        gcm_aes_xpn_128:
+          description: |-
+              GCM-AES-XPN-128.
+          type: boolean
+          default: true
+          x-field-uid: 3
+        gcm_aes_xpn_256:
+          description: |-
+              GCM-AES-XPN-256.
+          type: boolean
+          default: true
+          x-field-uid: 4
+
+    Mka.Basic.KeySource:
+      description: >-
+        The container for static key settings.
+      type: object
+      properties:
+        choice:
+          description: |-
+              Key source. Choose one from PSK or MSK.
+          type: string
+          default: psk
+          x-field-uid: 1
+          x-enum:
+            psk:
+              x-field-uid: 1
+            msk:
+              x-field-uid: 2
+        psk_chain:
+          description: |-
+            PSK chain.
+          $ref: '#/components/schemas/Mka.Basic.KeySource.PskChain'
+          x-field-uid: 2
+
+    Mka.Basic.KeySource.PskChain:
+      description: >-
+        The container for PSK chain settings.
+      type: object
+      properties:
+        name:
+          description: |-
+            PSK chain name.
+          type: string
+          x-field-uid: 1
+        psks:
+          description: |-
+            Pre-shared Keys.
+          type: array
+          items:
+            $ref: '#/components/schemas/Mka.Basic.KeySource.Psk'
+          x-field-uid: 2
+
+    Mka.Basic.KeySource.Psk:
+      description: >-
+        The container for Pre-shared key(PSK).
+      type: object
+      properties:
+        cak_value:
+          description: |-
+              CAK value.
+          type: string
+          format: hex
+          minLength: 16
+          maxLength: 32
+          default: "F123456789ABCDEF0123456789ABCDEF"     
+          x-field-uid: 1
+        cak_name:
+          description: |-
+              CAK name.
+          type: string
+          format: hex
+          minLength: 1
+          maxLength: 32
+          default: "F123456789ABCDEF0123456789ABCDEFF123456789ABCDEF0123456789ABCDEF"
```

Comment:
Are these defaults reasonable ? Will having them same on all interfaces be okay ? [ e.g. MAC or IP cannot be ]